packages/mdc-tabs/tab-bar/mdc-tab-bar.scss

```diff
@@ -41,10 +41,6 @@
   &__indicator {
     width: 100%;
 
-    @include mdc-theme-dark(".mdc-tab-bar") {
-      @include mdc-theme-prop(background-color, text-primary-on-dark);
```

Comment:
Can you replace all occurrences of `-on-light` with `-on-background` in this file?